it.json

```diff
@@ -83,6 +83,11 @@
                 "trigger" : "language",
                 "title" : "Scegli lingua",
                 "currentLanguage" : "Stai attualmente usando [language]."
```

Comment:
A comma is missing at the end of the line. Without it the file will break and can't be used.